webapp/src/translations/languages/en.json

```diff
@@ -914,38 +914,38 @@
     "importSuccess": "Wallet imported successfully",
     "masterNodeOperatorAddressFailure": "Masternode operator address is not present",
     "restartNode": "Restarting node. This may take a few minutes.",
-    "refreshUtxos": "Refreshing UTXOs it may take some time.",
+    "refreshUtxos": "Refreshing UTXOs. This may take some time.",
     "nodeDisconnected": "Node is disconnected",
     "electronRequiredError": "Electron app is needed for restart",
     "addressIsNotAPartOfWallet": "%{addressName} is not a part of wallet.",
     "timeoutMessage": "Timeout updated to %{timeout} seconds",
     "timeoutTitle": "Update Auto-lock Time",
     "updateAppNoticeTitle": "Update complete",
-    "updateAppNotice": "You need to restart DeFi Wallet to apply new update. Do you want to restart?",
+    "updateAppNotice": "You need to restart DeFi Wallet to apply the update. Do you want to restart?",
     "restartNowNotice": "Restart now",
     "restartLaterNotice": "Later",
     "showUpdateAvailableHeader": "An update is available for DeFi Wallet",
-    "showUpdateAvailableNotice": "Would you like to update? Kindly leave DeFi Wallet running for next 30 minutes, pruning work will take place and result in reduce storage required on your computer.",
+    "showUpdateAvailableNotice": "Would you like to update? Kindly leave DeFi Wallet running for the next 30 minutes, pruning work will take place and result in reduced storage required on your computer.",
     "yesShowUpdateAvailableButton": "Update",
     "noShowUpdateAvailableButton": "Not now",
     "startUpdateLabel": "Initiating update",
     "closeBtnLabel": "Close",
     "minimizeBtnLabel": "Minimize",
     "downloadingUpdate": "Downloading update",
     "reindexModelHeader": "Error occurred",
-    "restartAppWithReindexNotice": "Corrupted block database detected. You can restart app via node sync or snapshot download. Do you want to restart?",
+    "restartAppWithReindexNotice": "Corrupted block database detected. You can restart the app via node sync or snapshot download. Do you want to restart?",
     "yesRestartAppWithReindex": "Restart app",
     "noCloseApp": "Quit app",
-    "backupWalletNoticeTitle": "Backup wallet",
-    "backupWalletNotice": "Please backup your wallet before updating your app in order to prevent from any mishap. Do you want to backup your wallet?",
+    "backupWalletNoticeTitle": "Back up wallet",
```

Comment:
Used as a noun, keep it as "Backup wallet"

Reply:
I disagree. "Back up" is a verb here and really cannot be used as a noun. I suggest the wording "create backup" instead.

Reply:
Thanks for suggestion. Over here, it's used as a title in a popup to notify users. It's also used with consistency with "Backup wallet" button located at the menu bar.